apps/web/pages/settings/platform/oauth-clients/edit.tsx

```diff
@@ -0,0 +1,32 @@
+import Shell from "@calcom/features/shell/Shell";
+import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
+
+import PageWrapper from "@components/PageWrapper";
+import { EditOAuthClientForm } from "@components/settings/platform/oauth-clients/oauth-client-form/edit";
+
+export default function CreateOAuthClient() {
+  const searchParams = useCompatSearchParams();
+  const clientId = searchParams?.get("clientId") || "";
```

Comment:
I would use a dynamic param and reorganize, instead of getting the clientId from searchparams, use next.js routing 
and have something like this /platform/oauth-clients/:OAUTH_CLIENT_ID/edit